.nvmrc

```diff
@@ -1 +1 @@
-10
+14
```

Comment:
Seems that we need to use 14.5 or above to make it compatible with some module.

Reply:
```suggestion
14.18.2
```